examples/fractal.rs

```diff
@@ -45,8 +44,8 @@ fn main() {
 
 
     // Save the image as “fractal.png”
-    let fout = &mut File::create(&Path::new("fractal.png")).unwrap();
```

Comment:
This isn't really necessary, because of the `Into` trait.

Reply:
I'm not sure if I understand correctly, I deleted the `Path::new` since it is redundant. Are you saying you'd prefer to keep the `Path::new` there?

Reply:
Sorry, I was mixing the deleted lines with added ones.